Description: Fix brackets in Makefile thst prepare to run sphine.
Author: Andrew Lee (李健秋) <ajqlee@debian.org>
--- open-build-service-2.7.1.orig/src/api/Makefile
+++ open-build-service-2.7.1/src/api/Makefile
@@ -14,7 +14,8 @@ prepare_dirs:
 	$(INSTALL) -d -m 755 $(DESTDIR)$(OBS_API_PREFIX)/tmp
 	$(INSTALL) -d -m 755 $(DESTDIR)$(OBS_API_PREFIX)/config
 	# prepare for running sphinx daemon
-	$(INSTALL) -m 0755 -d $(DESTDIR)$(OBS_API_PREFIX)/db/sphinx{,/production}
+	$(INSTALL) -m 0755 -d $(DESTDIR)$(OBS_API_PREFIX)/db/sphinx
+	$(INSTALL) -m 0755 -d $(DESTDIR)$(OBS_API_PREFIX)/db/sphinx/production
 
 prepare_rake: prepare_dirs
 	cp -a * $(DESTDIR)$(OBS_API_PREFIX)
